include/wx/textctrl.h

```diff
@@ -517,10 +539,13 @@ class WXDLLIMPEXP_CORE wxTextAttr
     wxString            m_urlTarget;
     wxFontEncoding      m_fontEncoding;
 
+    long                m_paragraphUserData;
+
     // Character styles
     wxColour            m_colText,
                         m_colBack;
-    int                 m_fontSize;
+    float               m_fontSize;
+    int                 m_PixelSize;
```

Comment:
```suggestion
    int                 m_pixelSize;
```
I can do it myself when applying, but the code in wx should follow the `camelCase` naming convention (even with an extra `m_` prefix).